app/src/main/java/com/github/se/polyfit/di/UserModule.kt

```diff
@@ -90,16 +89,6 @@ object UserModule {
     return PostFirebaseRepository()
   }
 
-  @Provides
-  @Singleton
-  fun provideAuthentication(
-      @ApplicationContext context: Context,
-      user: User,
-      userFirebaseRepository: UserFirebaseRepository
-  ): AuthenticationCloud {
-    return AuthenticationCloud(context, user, userFirebaseRepository)
-  }
-
```

Comment:
Potential issue I just realized: the first time the user signs into the app, because the value of `user` is set to `"test"` when Firebase is initialized, the meals will be stored there instead of in the current user's folder. Do you think this could potentially cause problems ?